datahub-web-react/src/components/browse/BrowseResultCard.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Card } from 'antd';
+import { Link } from 'react-router-dom';
+
+type Props = {
+    url: string;
+    name: string;
+    count?: number | undefined;
+};
+
+export default function BrowseResultCard({ url, count, name }: Props) {
```

Comment:
Really like pulling this out into a separate component. We should do the same for Search Results!

Reply:
yep- ill put it on my todo list with the filters > url params refactor